src/Homework4/05_jazyki.py

```diff
@@ -0,0 +1,38 @@
+"""
+Языки
+Каждый из N школьников некоторой школы знает Mi языков.
+Определите, какие языки знают все школьники и языки,
+которые знает хотя бы один из школьников.
+Входные данные
+Первая строка входных данных содержит количество школьников N.
+Далее идет N чисел Mi, после каждого из чисел идет Mi строк,
+содержащих названия языков, которые знает i-й школьник.
+Пример:
+3          - N количество школьников
+2          - M1 количество языков первого школьника
+Russian    - языки первого школьника
+English
+3          - M2 количество языков второго школьника
+Russian
+Belarusian
+English
+3
+Russian
+Italian
+French
+
+Выходные данные
+В первой строке выведите количество языков, которые знают все
+школьники. Начиная со второй строки - список таких языков.
+Затем - количество языков, которые знает хотя бы один школьник,
+на следующих строках - список таких языков.
+"""
+
+langs = {}
+for i in range(int(input('количество школьников? '))):
+    langs[i] = set()
+    for j in range(int(input('количество языков школьника? '))):
+        langs[i].add(input('язык? '))
+al_ = set.intersection(*langs.values())
+one_ = set.union(*langs.values())
+print(len(al_), *list(al_), len(one_), *list(one_), sep='\n')
```

Comment:
it would be nice if we'll add description what is what :)

Reply:
added